samples/assistant/README.md

```diff
@@ -162,25 +162,25 @@ Also note that the storage of chat history is done via table storage. You may co
 1. Ask the assistant to create a todo item by sending a `POST` request to the `PostUserQuery` function to send a prompt to the assistant. The following is an example request:
 
     ```http
-    POST http://localhost:7168/api/assistants/assistant123
-    Content-Type: text/plain
+    ### Reminder #1 - Remind me to call my dad
+    POST http://localhost:7071/api/assistants/assistant123?message=Remind%20me%20to%20call%20my%20dad
 
-    Remind me to call my dad
     ```
 
-    The response should be an HTTP 202, indicating that the request was accepted. You should also see some relevant log output in the terminal window where the app is running.
+    The response should be an HTTP 200 and something like below:
 
-    > **NOTE:** The AI assistant runs asynchronously in the background, so it doesn't respond immediately to prompts.  You should wait for about 10 seconds before sending subsequent messages to the assistant to give it time to finish processing the previous prompt.
+    ```json
+    "I've added \"Call my dad\" to your todo list."
```

Comment:
It would be better if the response came as `text/plain` instead of JSON. If you think it should be JSON, then it should come back as a JSON object rather than as a naked JSON-encoded string.

Reply:
text/plain makes more sense. updated to it.